src/main/java/com/minecolonies/network/messages/WorkOrderChangeMessage.java

```diff
@@ -0,0 +1,116 @@
+package com.minecolonies.network.messages;
+
+import com.minecolonies.colony.Colony;
+import com.minecolonies.colony.ColonyManager;
+import com.minecolonies.colony.buildings.AbstractBuilding;
+import io.netty.buffer.ByteBuf;
+import net.minecraftforge.fml.common.network.simpleimpl.IMessage;
+import net.minecraftforge.fml.common.network.simpleimpl.IMessageHandler;
+import net.minecraftforge.fml.common.network.simpleimpl.MessageContext;
+
+/**
+ * Creates the WorkOrderChangeMessage which is responsible for changes in priority or removal of workOrders.
+ */
+public class WorkOrderChangeMessage implements IMessage, IMessageHandler<WorkOrderChangeMessage, IMessage>
+{
+    /**
+     * The Colony ID;
+     */
+    private int colonyId;
+
+    /**
+     * The workOrder to remove or change priority.
+     */
+    private int workOrderId;
+
+    /**
+     * The priority.
+     */
+    private int priority;
+
+    /**
+     * Remove the workOrder?
+     */
+    private boolean removeWorkOrder;
+
+    /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)